go/libkb/leveldb.go

```diff
@@ -158,7 +158,14 @@ func (l *LevelDb) doWhileOpenAndNukeIfCorrupted(action func() error) (err error)
 	// without resetting `dbOpenerOcce` (i.e. next call into LevelDb would result
 	// in a LevelDBOpenClosedError), because if DB open fails, retrying it
 	// wouldn't help. We should find the root cause and deal with it.
-
+	// MM: 10/12/2017: I am changing the above policy. I am not so sure retrying it won't help,
```

Comment:
I'm curious what on iOS causes temporarily failure on open that's recoverable later on?